en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -14,8 +14,8 @@ We'll go through each of these in detail in the following sections.
 
 | Main interface | Function |
 | :--- | :--- |
-| POWER1 | Connect Power module;  power input & AD voltage  and current detection. |
-| POWER2 | Connect i2c smart battery. |
+| Power | Connect Power module;  power input & AD voltage  and current detection. |
+| PM2 | Connect i2c smart battery. |
 | TF CARD |Insert SD card for log storage|
```

Comment:
@cuhome - Do they need to insert the card on the nano, or is this pre-inserted in the factory as stated below (on nano)?

Reply:
This modification is very good.

Reply:
They don't need to insert the card into the nano because the factory has already inserted the card into the nano.